cmd/frontend/backend/repos.go

```diff
@@ -41,12 +41,12 @@ func (e ErrRepoSeeOther) Error() string {
 
 var Repos = &repos{
 	store: database.GlobalRepos,
+	cache: dbcache.NewCachedDefaultRepoLister(database.GlobalRepos),
```

Comment:
Feel free to ignore, but I just had the idea for this name:


```suggestion
	cache: cached.NewDefaultRepoLister(database.GlobalRepos),
```